data/upgrades/astromech.json

```diff
@@ -559,5 +559,53 @@
     ],
     "cost": { "value": 3 },
     "restrictions": [{ "factions": ["Scum and Villainy"] }]
+  },
+  {
+    "name": "Watchful Astromech",
+    "limited": 0,
+    "xws": "watchfulastromech",
+    "hyperspace": true,
+    "sides": [
+      {
+        "ability": "After you perform a [Reload] or [Rotate Arc] action, i you are in an enemy ship's firing arc, you may perform a red [Calculate]",
```

Comment:
Typo in ability, should be "After you perform a [Reload] or [Rotate Arc] action, if you are in an enemy ship's firing arc, you may perform a red [Calculate] action."